components/gallery/GalleryItemAction/GalleryItemActionSection.vue

```diff
@@ -3,7 +3,7 @@
     <div>
       <p class="has-text-grey">{{ title }}</p>
       <div class="is-flex is-align-items-center">
-        <div v-if="Number(price)" class="has-text-weight-bold is-size-3 mr-5">
+        <div v-if="Number(price)" class="has-text-weight-bold is-size-3 mr-6">
```

Comment:
I'm about to remove `mr-5` here because it's incompatible with mobile

Reply:
wouldnt it break the web version then?

Reply:
No, I would put the margin style in the custom css accordingly.